Controller/CRUDController.php

```diff
@@ -221,7 +221,8 @@ public function deleteAction($id)
     /**
      * Edit action.
      *
-     * @param int|string|null $id Optional ID to use instead of globally provided default.
+     * @param int|string|null $id Optional ID representing the main admin subject.
```

Comment:
Then 1fabea3a7b422742dea6f87ab1265153fd00c268 was created, and it was not used anymore. IMO, this parameter should be deprecated and remove, unless we find a very good reason for it. Maybe you have one @curry684 ?